compiler/integration-tests/deploy-verifiers.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd ../..
+
+anvil > /dev/null 2>&1 &
```

Comment:
This causes us to have persistent anvil instance running in the background until the developer kills it in the task manager. This results in mysterious errors if they rerun the script as the port which anvil wants to use will already be occupied.

We should bring anvil back to the foreground when running locally so that it will close once the developer `ctrl-c`s out of this script.

Reply:
Addressed in [6b8abbc](https://github.com/noir-lang/noir/pull/2890/commits/6b8abbc1b86ee079b469d80531372a45c00bedfc)